### PR TITLE
Fixes #8327 push commit does clear commit message

### LINKED ISSUE #8327: Push commit does clear commit message when confirm box appears
## Current behaviour

Click on push clears the commit message input, while push confirmation box appears.

## Expected behaviour

Do not clear the commit message input.

## Steps to reproduce

Go to commit and type a commit message and click on commit and push.

## Environment

- GitExtensions version: 3.4.2
- GIT version: 2.23.0
- OS version:  Windows 10

### PR DESCRIPTION
**Pull request by racecrew**
Fixes #8327 

This PR is related to closed one #8513. All related PR description can be found there.

### CODE REVIEW

File: GitUI/CommandsDialogs/FormCommit.cs [hunk 1]
```diff
@@ -1484,15 +1484,22 @@ void DoCommit()
 
                     ScriptManager.RunEventScripts(this, ScriptEvent.AfterCommit);
 
-                    Message.Text = string.Empty; // Message.Text has been used and stored
+                    // Message.Text has been used and stored
                     _commitMessageManager.ResetCommitMessage();
                     CommitKind = CommitKind.Normal;
```
Comment (mstv) [inline GitUI/CommandsDialogs/FormCommit.cs:1490]: Remove this empty line, please, in order to indicate - at least a little - that `Message.Text = string.Empty` belongs to the comment above the block.

Comment (racecrew) [reply GitUI/CommandsDialogs/FormCommit.cs]: Done!

**Review by RussKie — APPROVED** [review]

**Comment by RussKie** [comment]
@msftbot merge if @mstv approves

**Review by mstv — APPROVED** [review]
Thank you

**Pull request merged**